packages/internal/src/build/babel/web.ts

```diff
@@ -11,12 +11,16 @@ import {
   getCommonPlugins,
   registerBabel,
   RegisterHookOptions,
+  parseConfigFiles,
+  getPathsFromConfig,
 } from './common'
 
 export const getWebSideBabelPlugins = (
   { forJest, forVite }: Flags = { forJest: false, forVite: false }
 ) => {
   const rwjsPaths = getPaths()
+  //get the config object from the file
```

Comment:
Let's improve this comment a bit:

```suggestion
  // Get the TS configs in the api and web sides as an object
```

Reply:
I'm glad to change this, but the config file can also be a `jsconfig.json` should we still change it to TS configs?

Reply:
You're right but I think it’s ok to refer to it as TS config; they’re both files for configuring TypeScript, so I think TS config gets it across

Reply:
perfect, thanks for the clarification!